src/axom/core/docs/sphinx/core_acceleration.rst

```diff
@@ -0,0 +1,51 @@
+.. ## Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level LICENSE file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+******************************************************
+Core acceleration
+******************************************************
+
+Axom Core provides support to ease the implementation of user applications that
+intend to support execution utilizing accelerators. In order to supply this,
+Axom provides a simple API to access memory and computation, using RAJA
+and Umpire to access accelerators.
+
+.. warning:: 
+   You do need to compile with RAJA and Umpire to fully leverage this Axom functionality. 
+   While both submodules will operate without these tools, they will offer little.

```

Comment:
It's not clear to me what you mean by "submodules" here.

Reply:
rephrased